### DISCUSSION TITLE
Use Both Plane (Ground & Vertical wall) for Tracking and model placement

### DISCUSSION THREAD

**Discussion by shubhamraghav1** [google/model-viewer#3093] [Q&A]
Hi..
Can anyone suggest me how do i use both ground plane and vertical plane tracking at same time to place 3d model at its junction like bottom of model on ground and backside of model on vertical plane

**Comment by elalish** [comment]
We don't support that currently. Honestly, given the reliability of AR tracking, I doubt it would work well in practice.

**Reply by shubhamraghav1** [reply]
But is there any chance to modify the code for this specific purpose, with some R&D

**Reply by elalish** [reply]
Certainly; just fork us and  tweak `ARRenderer.ts`.
